bot/resources/tags/blocking.md

```diff
@@ -0,0 +1,28 @@
+**Why do we need asynchronous programming?**
+
+Imagine that you're coding a Discord bot and every time somebody uses a command, you need to get some information from a database. But there's a catch: the database servers are acting up today and take a whole 10 seconds to respond. If you did **not** use asynchronous methods, your whole bot will stop running until it gets a response from the database. How do you fix this? Asynchronous programming.
```

Comment:
> If you did **not**

and then later on

> your whole bot will

This should be in the same tense, either past or present, but not both

Reply:
The rest of the tag seems to be present tense, so I'd recommend changing `if you did` to `if you do`